containerd/README.md

```diff
@@ -0,0 +1,87 @@
+# containerd Integration
+
+## Overview
+
+[...]
+
+* You can also use images here:
+
+![snapshot][1]
+
+## Setup
+
+### Installation
+
+[...]
```

Comment:
Need help filling this in?

Reply:
I will take care of this later this week. I wanted to submit the core of the contrib today. Having a review from your team once ready would be great.